### PR TITLE
ci: Update Go versions

### PR DESCRIPTION
**Pull request by francislavoie**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -108,6 +108,19 @@ jobs:
...
+        shell: bash
+        run: |
+          printf "Using go at: $(which go)\n"
+          printf "Go version: $(go version)\n"
+          printf "\n\nGo environment:\n\n"
+          go env
+          printf "\n\nSystem environment:\n\n"
+          env
+          # Calculate the short SHA1 hash of the git commit
+          echo "short_sha=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
```
Comment (mohammed90) [inline .github/workflows/ci.yml:122]: Good catch 👍🏼

**Comment by francislavoie** [comment]
Hmm... why didn't CI run??

**Review by mohammed90 — APPROVED** [review]
Thanks for doing this, Francis!

**Comment by francislavoie** [comment]
I'm confused. It's not producing actions.

**Comment by francislavoie** [comment]
but the PR is to `master`... it's always worked :grimacing:

**Comment by mohammed90** [comment]
> but the PR is to `master`... it's always worked 😬

True. I'm as confused as you are.

**Comment by francislavoie** [comment]
Bleh. I guess we could just merge. It _should_ run on `master`. If an error happens I'll just do another PR to fix it.

**Comment by francislavoie** [comment]
:man_shrugging: it ran and passed. Whatever.

**Pull request merged**